procedures/unit-testing-basics.ipf

```diff
@@ -3447,6 +3453,13 @@ Function RunTest(procWinList, [name, testCase, enableJU, enableTAP, enableRegExp
 
 	else
 		// no early return/abort above this point
+		if(!!fixLogName)
+			dfref dfr = GetPackageFolder()
+			string/G dfr:baseFilenameOverwrite = "IUTF_Test"
+		else
+			dfref dfr = GetPackageFolder()
+			KillVariables/Z dfr:baseFilenameOverwrite
```

Comment:
See my comment above. Make sure you have only a single location where you create baseFilenameOverwrite through
`string/G dfr:baseFilenameOverwrite`